doc/source/release-notes/Percona-Server-5.7.26-29.rst

```diff
@@ -0,0 +1,105 @@
+.. rn:: 5.7.26-29
+	
+==========================
+|Percona Server| |release|
+==========================
+	
+Percona is glad to announce the release of |Percona Server| |release| on
+May 22, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.26-29/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.26
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-26.html>`_
+and includes all the bug fixes in it. |Percona Server| |release| is
+now the current GA (Generally Available) release in the 5.7 series.
+	
+New Features
+============
+
+- New :variable:`Audit_log_buffer_size_overflow` status variable has been
+  implemented to track when an :ref:`audit_log_plugin` entry was either
+  dropped or written directly to the file due to its size being bigger
+  than :variable:`audit_log_buffer_size` variable.
+
+Bugs Fixed
+==========
+
+- TokuDB storage engine would assert on load when used with jemalloc 5.x.
+  Bug fixed :psbug:`5406`
+
+- a read-write workload on compressed InnoDB tables could cause an assertion
+  error. Bug fixed :psbug:`3581`.
+	
+- using TokuDB or MyRocks native partitioning and ``index_merge`` could lead to
+  a server crash. Bugs fixed :psbug:`5206`, :psbug:`5562`.
+
+- a stack buffer overrun could happen if the redo log encryption with
+  key rotation was enabled. Bug fixed :psbug:`5305`.
+
+- TokuDB and MyRocks native partitioning handler objects were allocated from a 
+  wrong memory allocator. Memory was released only on shutdown and concurrent
+  access to global memory allocator caused memory corruptions and therefore
+  crashes. Bugs fixed :psbug:`5508`, :psbug:`5525`.
+
+- redo log encryption wasn't working. Bug fixed :psbug:`5547`. 
```

Comment:
Needs more details. Enabling redo log encryption resulted in redo log being written unencrypted or similar